test/box-luatest/gh_6930_mvcc_isolation_levels_test.lua

```diff
@@ -0,0 +1,140 @@
+local server = require('test.luatest_helpers.server')
```

Comment:
I don't like the amount of code duplication in the tests — could we merge them, factor the common parts into separate functions and then test the two patches together in different subtests?